app/Http/Controllers/SettingsController.php

```diff
@@ -1433,26 +1352,30 @@ public function api()
      * Test the email configuration.
      *
      * @author [A. Gianotto] [<snipe@snipe.net>]
-     *
      * @since [v3.0]
-     *
-     * @return \Illuminate\Http\RedirectResponse
      */
-    public function ajaxTestEmail()
+    public function ajaxTestEmail() : JsonResponse
     {
         try {
             (new User())->forceFill([
                 'name'  => config('mail.from.name'),
                 'email' => config('mail.from.address'),
             ])->notify(new MailTest());
 
-            return response()->json(Helper::formatStandardApiResponse('success', null, 'Maiol sent!'));
```

Comment:
Wow, we had that in there? Sigh. Thank you.

Reply:
Yep. :(